README.md

```diff
@@ -22,9 +21,71 @@ Or install it yourself as:
 
     $ gem install microform
 
+## Generating Forms
+
+Generate a form for your model, with accompanying tests:
+
+```
+  rails g microform:form model_name
+```
+
+This will create two files: `app/forms/model_name_form.rb` and `test/forms/model_name_test.rb`.
+
 ## Usage
 
-TODO: Write usage instructions here
+ The `Microform::Submission` module provides a `submit` method that will help to better isolate your form tests. Include it in the relevant controller for your form(s):
+
+```ruby
+  class AdminController < ApplicationController
```

Comment:
Can you change this to `class ApplicationController < ActionController::Base` in order to use an example that is more default-y (since `AdminController` isn't generated in a new Rails project).